private-resources/META-INF/maven/gov.usdot/fedgov-cv-private-resources-webapp/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>fedgov-cv-private-resources-webapp</artifactId>
+	<packaging>war</packaging>
+	<name>Connected Vehicle Private Resources WebApp</name>
+	<description>A webapp providing private resources to the other CV webapps</description>
+	<parent>
+		<groupId>gov.usdot</groupId>
+		<artifactId>fedgov-cv-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+		<relativePath>../fedgov-cv-parent/pom.xml</relativePath>
+	</parent>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<java.version>1.8</java.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.jasig.cas</groupId>

```

Comment:
Is this dependency required?

Reply:
Yes, functionality was broken when it was removed